src/instance.rs

```diff
@@ -1243,24 +1270,16 @@ impl<T> Py<T> {
     ///
     /// # Safety
     /// `ptr` must point to a Python object of type T.
-    #[inline]
```

Comment:
Removed `inline` here because it's only used in this module, I figure it'll be inlined anyway.

Reply:
While it is available for inlining (just because it is in the same crate), `#[inline]` will still increase the likelyhood that it actually is inlined, i.e. it will bias the heuristics towards using that option, i.e. I would not remove it since this should be compiled away completed only after inlining which should happen immediately in the first pass of LLVM over the resulting MIR.